schema_salad/sourceline.py

```diff
@@ -15,19 +17,19 @@ def _add_lc_filename(r, source):  # type: (ruamel.yaml.comments.CommentedBase, A
         for d in r:
             _add_lc_filename(d, source)
     elif isinstance(r, dict):
-        for d in r.itervalues():
+        for d in six.itervalues(r):
             _add_lc_filename(d, source)
 
-def relname(source):  # type: (AnyStr) -> AnyStr
```

Comment:
Any particular reason this was marked as ``AnyStr``? I don't expect this to take binary strings.